src/main/java/Greet.java

```diff
@@ -0,0 +1,102 @@
+import java.util.ArrayList;
+
+public class Greet {
```

Comment:
Perhaps the class name should be a noun, as per [coding standards](https://se-education.org/guides/conventions/java/basic.html#naming)? `Ui` could be an even better name, you can take a look at Address Book for examples.